src/components/MediaCounterButton/MediaCounterButton.story.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withDocs } from 'storybook-readme';
+import { select, number } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import MediaCounterButton from './';
+import README from './README.md';
+import theme from 'theme';
+
+const variants = Object.keys(theme.mediaCounter); // dark, light
+
+storiesOf('Components/MediaCounterButton', module)
+  .addDecorator(withDocs(README))
+  .add('default', () => (
+    <MediaCounterButton
+      variant={select('Variant', variants, variants[0])}
```

Comment:
Nitpicking: should we leave it to the default value?

```suggestion
      variant={select('Variant', variants)}
```